tests/data/json/embed.json

```diff
@@ -8,7 +8,7 @@
     "thumbnail_url": "https://www.imdb.com/title/tt8936646/mediaviewer/rm2618735617?ref_=tt_ov_i",
     "thumbnail_width": 290,
     "thumbnail_height": 234,
-    "caption": "Extraction",
```

Comment:
@Pradhvan don't edit existing valid testcases. we should have test cases for all conditions.